praw/reddit.py

```diff
@@ -272,9 +277,32 @@ def _prepare_objector(self):
         for kind, klass in mappings.items():
             self._objector.register(kind, klass)
 
-    def _prepare_prawcore(self):
-        requestor = Requestor(USER_AGENT_FORMAT.format(self.config.user_agent),
-                              self.config.oauth_url, self.config.reddit_url)
+    def _prepare_prawcore(self, requestor_class=None, requestor_kwargs=None):
+        requestor_class = requestor_class or Requestor
+        requestor_kwargs = requestor_kwargs or {}
+
+        # Allow overriding oauth_url and reddit_url as Requestor keywords
+        oauth_url = requestor_kwargs.pop('oauth_url', None)
```

Comment:
Let's not make these values overwritable. If someone really want to change them, they can do so within the Requestor itself, but they're already changeable via the PRAW's configuration (praw.ini, env, or as arguments to `Reddit` itself.